include/RemotePluginClient.h

```diff
@@ -126,6 +135,47 @@ class RemotePluginClient : public RemotePluginBase
 	fpp_t m_bufferSize;
 } ;
 
+#ifndef LMMS_BUILD_WIN32
+class PollParentThread
+{
+public:
+	PollParentThread() :
+		m_stop{false},
+		m_thread{
+			[this]
+			{
+				using namespace std::literals::chrono_literals;
+				auto lock = std::unique_lock{m_mutex};
+				while (!m_cv.wait_for(lock, 500ms, [this] { return m_stop; }))
+				{
+					if (getppid() == 1)
```

Comment:
I can't really find any documentation on how `getppid` is supposed to work on Windows. Maybe a comment on why this should `== 1`?

Reply:
Because this code isn't for Windows.

Reply:
ah crap, I always miss that stupid `n` in the `ifndef`. Ignore me. 😩